src/content/guides/merchant-integration-qr-code-flow.mdoc

```diff
@@ -40,10 +38,9 @@ sequenceDiagram
 
 1. The POS [creates a Payment Request](/api/payment-requests#create-a-payment-request) and presents a QR code to the Patron on a customer-facing display.
 
-  > The QR code decodes to a URL of the form `https://service.centrapay.com/api/payment-requests/{paymentRequestId}/pay`
+    > The QR code decodes to a URL of the form `https://service.centrapay.com/api/payment-requests/{paymentRequestId}/pay`
```

Comment:
@tom-blake Question: I know you're refactoring but is this correct? 🤨 
It normally decodes to `https://app.centrapay.com/pay/{paymentRequestId}` - is it different for POS integrations?